flycheck.el

```diff
@@ -651,6 +651,17 @@ using \\[flycheck-error-list-reset-filter]."
   :safe #'flycheck-error-level-p
   :package-version '(flycheck . "0.24"))
 
+(defcustom flycheck-relevant-error-other-file-min-level 'error
+  "The minimum level of errors from other files to display in this buffer.
+
+If set to an error level, only display errors from other files
+whose error level is at least as severe as this one list.  If
+nil, display all errors from other files."
+  :group 'flycheck
+  :type 'flycheck-minimum-level
+  :safe #'flycheck-error-level-p
+  :package-version '(flycheck . "32"))
```

Comment:
Should this be documented in the manual, too?

Reply:
Good point.  I've added a section on "errors from other files" with this new variable.